src/udev/udevadm-trigger.c

```diff
@@ -304,6 +305,8 @@ int trigger_main(int argc, char *argv[], void *userdata) {
                 { "sysname-match",         required_argument, NULL, 'y'                       },
                 { "name-match",            required_argument, NULL, ARG_NAME                  },
                 { "parent-match",          required_argument, NULL, 'b'                       },
+                { "parent-ignore",         no_argument,       NULL, 'i'
+      },
```

Comment:
Spurious line break.
Also, maybe better to name the option `"include-parents"` or so. At least, `"parent-ignore"` sounds spurious and quite ambiguous.

Reply:
yes, solved